code/WorkInProgress/TempEngine.dm

```diff
@@ -13,6 +13,8 @@
 
 	var/side = null // 1=left 2=right
 	var/last_pressure_delta = 0
+	var/fan_efficiency = 10 // 0.9 ideal, but I don't want everyone to suffer... yet.
```

Comment:
0.9 or 9?

Reply:
0.9 is efficient for a fan with perfect being perfect FAN would be 1
This is a super duper fan but was a compromise to scale based pump.dm to not punish players until behavior was better understood.